modules/nf-core/gprofiler2/gost/main.nf

```diff
@@ -28,4 +28,24 @@ process GPROFILER2_GOST {
 
     script:
     template 'gprofiler2_gost.R'
+
+    stub:
+    def prefix = task.ext.prefix ?: meta.id
+    """
+    touch ${prefix}.gprofiler2.all_enriched_pathways.tsv
+    touch ${prefix}.gprofiler2.gost_results.rds
+    touch ${prefix}.gprofiler2.gostplot.png
+    touch ${prefix}.gprofiler2.gostplot.html
+    touch ${prefix}.gprofiler2.*.sub_enriched_pathways.tsv
+    touch ${prefix}.gprofiler2.*.sub_enriched_pathways.png
+    touch ${prefix}.ENSG_filtered.gmt
+    touch ${prefix}.R_sessionInfo.log
+
+    cat <<-END_VERSIONS > versions.yml
+    "${task.process}":
+        r-base: \$(echo \$(R --version 2>&1) | sed 's/^.*R version //; s/ .*\$//')
```

Comment:
We should actually change this, and the template, to not report the r version. This breaks every time Bioconda updates its R (which I only came to appreciate later on). 

In general we should only pin the main dependency, and only report the version of that dependency.